README.md

```diff
@@ -320,7 +320,7 @@ The `data` attribute provides an array of events for the period defined by the `
 * **Application {x} is not configured as a multi-tenant application.**
   * In your azure app go to Manifest, find the key "signInAudience", change its value to "AzureADandPersonalMicrosoftAccount"
 * **Platform error sensor.office365calendar - No module named '{x}'**
-  * This is a known home assistant issue, all that's needed to fix this should be another restart of your home assistant server. If this does not work, please try installing the component in this order:  
+  * This is a known home assistant issue, all that's needed to fix this should be another restart of your home assistant server. If this does not work, please try installing the component in this order:
```

Comment:
Looks good...maybe capitalize Home Assistant in both places?

Reply:
Oops, I thought you did that. Did I undo it?

Reply:
I think it was a couple I missed. I just didn't want to do another final final final. LOL.

You may want to search for home assistant (lowercases) and fix all of them.

Also, I just went and reconfigured my integration just so I didn't have to remember to do it later. In the process, I discovered two things using the new default auth method:

1. After the first HA restart, I was presented with an option to approve what looked like all or most of the permissions, but the calendar events did not populate. I restarted HA again and was presented with another persistent notification. This time, I only had to authorize e-mail read/write (I think). After that, the calendar started working again as before. It may be worth a mention - that it may take two restart/authentication cycles for the authentication to complete.
2. Now that I understand what I'm actually accomplishing with setting the API permissions, some of them become redundant (e.g. read/write and read) depending on one's use case. I guess it's ok to keep unnecessary permissions, but not sure if that's worth a mention.